umm-spec-lib/test/cmr/umm_spec/test/migration/version/collection.clj

```diff
@@ -3835,3 +3835,88 @@
+                                        {:Type "Field Campaign"}]
+                :DOI                   {:DOI "10.5067/fake.record.01"}
+                :SpatialExtent         {:GranuleSpatialRepresentation "NO_SPATIAL"}
+                :TemporalExtents       [{:EndsAtPresentFlag false
+                                         :SingleDateTimes   ["2024-02-14T01:02:03Z"]}]
+                :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.1",
+                                        :Name    "UMM-C",
+                                        :Version "1.18.1"}}
+               ;; sample-collection
+               {:AssociatedDOIs        [{:Type "IsPreviousVersionOf"}
+                                        {:Type "IsNewVersionOf"}
+                                        {:Type "Field Campaign"}]
+                :DOI                   {:DOI "10.5067/fake.record.01"}
+                :SpatialExtent         {:GranuleSpatialRepresentation "NO_SPATIAL"}
+                :TemporalExtents       [{:EndsAtPresentFlag false
+                                         :SingleDateTimes   ["2024-02-14T01:02:03Z"]}]
+                :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.2",
+                                        :Name    "UMM-C",
+                                        :Version "1.18.2"}}
+               )
+
+               "Migrating CollectionProgress enum PREPRINT back to COMPLETED"
+               ;; expected
+               {:CollectionProgress    "PLANNED"
+                :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.1",
+                                        :Name    "UMM-C",
+                                        :Version "1.18.1"}}
+                ;; sample-collection
+               {:CollectionProgress    "PREPRINT"
+                :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.2",
+                                        :Name    "UMM-C",
+                                        :Version "1.18.2"}}
+
+               "Migrating CollectionProgress enum PREPRINT back to COMPLETED"
+               ;; expected
+               {:CollectionProgress    "PLANNED"
+                :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.1",
+                                        :Name    "UMM-C",
+                                        :Version "1.18.1"}}
+               ;; sample-collection
+               {:CollectionProgress    "INREVIEW"
+                :MetadataSpecification {:URL     "https://cdn.earthdata.nasa.gov/umm/collection/v1.18.2",
+                                        :Name    "UMM-C",
+                                        :Version "1.18.2"}}
+
+               "Migrating CollectionProgress enum SUPERSEDED back to COMPLETED"
+               ;; expected
+               {:CollectionProgress    "PLANNED"
```

Comment:
SUPERSEDED should translate back to COMPLETED.

Reply:
done